sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1179,6 +1179,59 @@ def max_deflection(self):
         else:
             return None
 
+
+    def bending_stress(self, y, m):
+        b_stress = (m*y)/self.second_moment
+        return b_stress
+
+
+    def max_bending_stress(self, y_max, m=None):
+        """Returns the maximum bending stress and its coordinates
+        in the Beam object.
+
+        Parameters
+        ==========
+        y_max : Sympifiable
+                A Symbol or a value for the maximum distance from the neutral axis
```

Comment:
This is not a typical thing to do in SymPy code. What is the purpose for the user to provide this? Can't they name it later if they want to?

Reply:
> What is the purpose for the user to provide this?

Here `y_max` has to be given by the user as a numerical value or else can be replaced by a symbol.

So, one way that I tried was to set a default value for `y_max` to `None` and to replace it with a `Dummy` if it remains `None` i.e. if no numerical value is being assigned. 

The other way that I have tried (and done) is to use it similar to how  `length` and `elastic_modulus` are used in the beam class, [here.](https://github.com/sympy/sympy/blob/master/sympy/physics/continuum_mechanics/beam.py#L83) 

I am not so sure about it, but the second one looked better.

>Can't they name it later if they want to?

I guess I didn't get your point. Does that mean by using `replace`?

Reply:
Isn't the max stress computed by taking the derivative of shear or moment wrt to x, setting to zero, solving for x, and then calculating the max stress at that x?

Reply:
>Isn't the max stress computed by taking the derivative of shear or moment wrt to x

Yes, and this is done by the functions `max_shear_force()` and `max_bmoment()` and the stress functions just make use of it.